libs/blocks/merch-card/merch-card.js

```diff
@@ -154,6 +154,52 @@ const parseTwpContent = async (el, merchCard) => {
   }
 };
 
+const appendPaymentDetails = (element, merchCard) => {
+  if (element.firstChild.nodeType !== Node.TEXT_NODE) return;
+  const paymentDetails = createTag('div', { slot: 'payment-details' }, element.innerHTML);
```

Comment:
can you use `element` instead of `element.innerHTML` ?

Reply:
I'm passing `element.innerHTML` because I want to strip off the `<h6>` tag - semantically this text is not a heading, and I think there is no need to wrap this text in an additional tag. Is there any reason why we should change it to `element`?

Reply:
if you know that `element` is never an element with an even listener attached, that should be ok, if not, this will cause the listeners to be removed from the element.